lib/mshoplib/src/MShop/Customer/Item/Standard.php

```diff
@@ -164,6 +164,17 @@ public function setPassword( string $value ) : \Aimeos\MShop\Customer\Item\Iface
 		return $this->set( 'customer.password', $value );
 	}
 
+    /**
+     * Verifies a given password to the current one
+     *
+     * @param string $value
+     * @return bool
+     */
+	public function verifyPassword( string $value ) : bool
+    {
+        return password_verify( $value, $this->getPassword());
```

Comment:
This does only work if the `password_hash()` method is used but will fail if `sodium_crypto_pwhash_str_verify()` or something else is used instead. For TYPO3, we make the password hashing object available in the context:
https://github.com/aimeos/ai-typo3/blob/master/lib/custom/src/MShop/Context/Item/Typo3.php

Think, we need the common "password" object now in the context which offers methods like `encode()` and `verify()` and forward the requests to the concrete implementation:
https://github.com/aimeos/aimeos-core/tree/master/lib/mshoplib/src/MShop/Common/Helper/Password